tests/acl_loader_test.py

```diff
@@ -250,7 +273,7 @@ def ttest_icmp_fields_with_non_icmpv6_protocol(self, acl_loader):
         assert not acl_loader.rules_info.get("RULE_1")
 
 
-    def test_icmp_fields_with_non_tcp_protocol(self, acl_loader):
+    def test_tcp_fields_with_non_tcp_protocol(self, acl_loader):
```

Comment:
why change this name?

Reply:
Because it's not an `ICMP` related test. I'm fixing this typo.